src/sass/pages/_3d-printing.scss

```diff
@@ -2,6 +2,7 @@
 .model-examples__title {
   margin-bottom: 16px;
 
+  font-family: Cera-Pro, sans-serif;
```

Comment:
We dont need this, because font-family added in _base.scss

Reply:
Modern-normalize break font-family styles of body

Reply:
Links order fixed. Please fix commit.

Reply:
Done